deployments/examples/ocis_full/.env

```diff
@@ -114,6 +114,23 @@ SMTP_AUTHENTICATION=
 # Allow insecure connections to the SMTP server. Defaults to false.
 SMTP_INSECURE=
 
+## oCIS Web Extensions ##
+# It is possible to use the oCIS Web Extensions to add custom functionality to the oCIS frontend.
+# For more details see https://github.com/owncloud/web-extensions/blob/main/README.md
+# Enable this to create a new named volume
+#EXTENSIONS=:web_extensions/extensions.yml
+# Enable the desired extensions by uncommenting the following lines.
+# Note: the leading colon is required to enable the service.
+# Note: if you want to remove a web extension, please delete the ocis-apps volume and restart the docker-compose.
```

Comment:
```suggestion
# Note: if you want to remove a web extension, you must delete the ocis-apps volume. It will be properly recreated on docker compose startup.
```